[33mWARNING [0m urllib3.connectionpool:connectionpool.py:750 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000003CCBAC8>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/5491d77ef1d32a15f90de74c6b98a553
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:750 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000003D44748>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/5491d77ef1d32a15f90de74c6b98a553
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:750 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000003D44E48>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/5491d77ef1d32a15f90de74c6b98a553